Game/Config/DefaultEditorPerProjectUserSettings.ini

```diff
@@ -1,2 +1,36 @@
 [ContentBrowser]
-ContentBrowserTab1.SelectedPaths=/Game/ThirdPersonCPP
\ No newline at end of file
+ContentBrowserTab1.SelectedPaths=/Game/ThirdPersonCPP
+
+[/Script/SpatialGDKEditorToolbar.SpatialGDKEditorToolbarSettings]
+ProjectRootFolder=(Path="C:/work/UnrealGDKTestSuite/spatial/")
```

Comment:
Maybe exclude the settings with the path so it resets to the default one? (since this path is different for everyone)